QSB/TornadoSync/TornadoManager.cs

```diff
@@ -0,0 +1,49 @@
+﻿using QSB.TornadoSync.TransformSync;
+using QSB.TornadoSync.WorldObjects;
+using QSB.Utility;
+using QSB.WorldSync;
+using QuantumUNET;
+
+namespace QSB.TornadoSync
+{
+	public class TornadoManager : WorldObjectManager
+	{
+		protected override void RebuildWorldObjects(OWScene scene)
+		{
+			QSBWorldSync.Init<QSBTornado, TornadoController>();
+
+			if (!QSBCore.IsHost)
+			{
+				return;
+			}
+
+			foreach (var transformSync in QSBWorldSync.GetUnityObjects<OccasionalTransformSync>())
+			{
+				QNetworkServer.Destroy(transformSync.gameObject);
+			}
+
+			var gdBody = Locator._giantsDeep.GetOWRigidbody();
+			// cannon
```

Comment:
why is the cannon in the same file as the islands/tornadoes ?

Reply:
idk, it's a giant deep thing and i wanted to sync it in a similar way for fun